frontend/src/store/facilityBooking/types.ts

```diff
@@ -88,6 +89,11 @@ export enum FACILITY_ACTIONS {
   SET_VIEW_FACILITY_NAME = 'FACILITY_ACTIONS.SET_VIEW_FACILITY_NAME',
   SET_CREATE_BOOKING_ERROR = 'FACILITY_ACTIONS.SET_CREATE_BOOKING_ERROR',
   SET_REPEAT_WEEKLY = 'FACILITY_ACTIONS.SET_REPEAT_WEEKLY',
+  SET_CONFLICT_BOOKING = 'FACILITY_ACTIONS.SET_CONFLICT_BOOKING',
+  SET_CONFLICT_BOOKING_STATUS = 'FACILITY_ACTIONS.SET_CONFLICT_BOOKING_STATUS',
+  SET_CONFLICT_BOOKING_START_TIME = 'FACILITY_ACTIONS.SET_CONFLICT_BOOKING_START_TIME',
+  SET_CONFLICT_BOOKING_END_TIME = 'FACILITY_ACTIONS.SET_CONFLICT_BOOKING_END_TIME',
+  SET_CONFLICT_BOOKING_END_DATE = 'FACILITY_ACTIONS.SET_CONFLICT_BOOKING_END_DATE',
```

Comment:
are these necessary?

Reply:
Resolved in commit ca5730df3da1e495b5afa9c949a610730cb97605.